doc/sphinx/references.bib

```diff
@@ -11590,6 +11590,22 @@ @Article{fraters:etal:2019
     eprint = {http://oup.prod.sis.lan/gji/article-pdf/218/2/873/28693654/ggz183.pdf},
 }
 
+@article{fraters:billen:2021,
```

Comment:
The keys in the dictionary in the make_cite_html.py file do not match this bibtex key. Use either ":" or "_" consistently.